token-syncer/integration/token_syncer/basic_test.clj

```diff
@@ -0,0 +1,411 @@
+;;
+;;       Copyright (c) 2017 Two Sigma Investments, LP.
+;;       All Rights Reserved
+;;
+;;       THIS IS UNPUBLISHED PROPRIETARY SOURCE CODE OF
+;;       Two Sigma Investments, LP.
+;;
+;;       The copyright notice above does not evidence any
+;;       actual or intended publication of such source code.
+;;
+(ns token-syncer.basic-test
+  (:require [clojure.string :as str]
+            [clojure.test :refer :all]
+            [clojure.tools.logging :as log]
+            [plumbing.core :as pc]
+            [token-syncer.main :as main]
+            [token-syncer.syncer :as syncer])
+  (:import (java.util UUID)))
+
+(def basic-description {"cmd" "echo 'Hello World'", "cpus" 1, "mem" 2048, "metric-group" "syncer-test"})
+
+(defn- waiter-urls []
+  (let [waiter-uris (System/getenv "WAITER_URIS")]
+    (is waiter-uris)
+    (-> waiter-uris
+        (str/split #",")
+        sort)))
+
+(defn- waiter-api []
+  (main/init-waiter-api {:connection-timeout-ms 5000, :idle-timeout-ms 5000}))
+
+(deftest ^:integration test-environment
+  (log/info "****** Running: test-environment")
```

Comment:
The leading `*`s are pretty distracting in my opinion

Reply:
Will do.

Reply:
Removed, changed them to comments.

Reply:
I still see them in the log line above

Reply:
Oops, removed that one too.